builders/integrations/indexers/covalent.md

```diff
@@ -46,70 +56,356 @@ If you're already familiar with Covalent and ready to dive in, you simply need t
+### Balances {: #balances }
 
-=== "Moonbase Alpha"
-    |                                                                                Endpoint                                                                                 |                                             Format                                              |
-    |:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------------:|
-    |          [Balances](https://www.covalenthq.com/docs/api/balances/get-token-balances-for-address/?utm_source=moonbeam&utm_medium=partner-docs){target=\_blank}           |      api.covalenthq.com/v1/{{ networks.moonbase.chain_id }}/address/{ADDRESS}/balances_v2/      |
-    |       [Transactions](https://www.covalenthq.com/docs/api/transactions/get-transactions-for-address/?utm_source=moonbeam&utm_medium=partner-docs){target=\_blank}        |    api.covalenthq.com/v1/{{ networks.moonbase.chain_id }}/address/{ADDRESS}/transactions_v2/    |
-    |                             [Transfers](https://www.covalenthq.com/docs/api/balances/get-erc20-token-transfers-for-address){target=\_blank}                             |     api.covalenthq.com/v1/{{ networks.moonbase.chain_id }}/address/{ADDRESS}/transfers_v2/      |
-    |                   [Token holders](https://www.covalenthq.com/docs/api/balances/#balances/get-token-holders-as-of-any-block-height-v2){target=\_blank}                   | api.covalenthq.com/v1/{{ networks.moonbase.chain_id }}/tokens/{CONTRACT_ADDRESS}/token_holders/ |
-    | [Log events (smart contract)](https://www.covalenthq.com/docs/api/base/get-log-events-by-contract-address/?utm_source=moonbeam&utm_medium=partner-docs){target=\_blank} |    api.covalenthq.com/v1/{{ networks.moonbase.chain_id }}/events/address/{CONTRACT_ADDRESS}/    |
-    |                            [Log events (topic hash)](https://www.covalenthq.com/docs/api/base/get-log-events-by-topic-hash){target=\_blank}                             |          api.covalenthq.com/v1/{{ networks.moonbase.chain_id }}/events/topics/{TOPIC}/          |
-    |                       [Security (Token Approvals)](https://www.covalenthq.com/docs/api/security/get-token-approvals-for-address){target=\_blank}                        |           api.covalenthq.com/v1/{{ networks.moonbase.chain_id }}/approvals/{ADDRESS}/           |
+???+ function "Token Balances"
+
+    The [token balances endpoint](https://goldrush.mintlify.app/docs/api/balances/get-token-balances-for-address){target=\_blank} retrieves native tokens, fungible (ERC20) tokens, and non-fungible (ERC721 & ERC1155) tokens associated with a given address. The returned data includes current market prices and additional token metadata.
```

Comment:
```suggestion
    The [token balances endpoint](https://goldrush.mintlify.app/docs/api/balances/get-token-balances-for-address){target=\_blank} retrieves native tokens, fungible (ERC20) tokens, and non-fungible (ERC721 and ERC1155) tokens associated with a given address. The returned data includes current market prices and additional token metadata.
```

Can you please check the rest of the docs to see if there should be a hyphen in ERC-20, etc.?